include/BSMPT/bounce_solution/bounce_solution.h

```diff
@@ -0,0 +1,557 @@
+   * false vacuum fraction drops below val
+   * @param false_vac_frac desired false vacuum fraction value
+   * @return temperature at which false vacuum fraction drops below val
+   */
+  double CalcTempAtFalseVacFraction(const double &false_vac_frac);
+
+  /**
+   * @brief CalcFalseVacFraction calculates false vacuum fraction as function of
+   * temperature
+   * @param temp temperature
+   * @return false vacuum fraction
+   */
+  double CalcFalseVacFraction(const double &temp);
+
+  /**
+   * @brief CalculatePercolationTemp calculation of the temperature when the
+   * false vacuum fraction drops below 71 % (default)
+   * @param false_vac_frac false vacuum fraction at percolation temperature, by
+   * default set to 71 %
+   */
+  void CalculatePercolationTemp(const double &false_vac_frac = 0.71);
+
+  /**
+   * @brief CalculateCompletionTemp calculation of the temperature when the
+   * false vacuum fraction drops below 1 % (default)
+   * @param false_vac_frac false vacuum fraction at completion temperature, by
+   * default set to 1 %
+   */
+  void CalculateCompletionTemp(const double &false_vac_frac = 0.01);
+
+  /**
+   * @brief Calculate phase transition strength alpha at percolation temperature
+   */
+  void CalculatePTStrength();
+
+  /**
+   * @brief Calculate wall velocity
+   * @param false_min initial, false minimum
+   * @param true_min final, true minimum
+   */
+  void CalculateWallVelocity(const Minimum &false_min, const Minimum &true_min);
+
+  /**
+   * @brief Calculate inverse time scale of phase transition
+   */
+  void CalculateInvTimeScale();
+
+  /**
+   * @brief Get inverse time scale of phase transition
+   */
+  double GetInvTimeScale();
+};
+
+/**
+ * @brief Nintegrate_Inner Numerical integration of inner integral over inverse
+ * Hubble rate for the percolation temperature calculation
+ * @param obj Class reference to pass all needed parameters
+ * @param T upper integration boundary
+ * @return Numerical value of integral and absolute error
+ */
+struct resultErrorPair Nintegrate_Inner(BounceSolution &obj,
+                                        const double &Tprime);
+
+/**
+ * @brief Nintegrate_Outer Numerical integration of outer integral for the
+ * percolation temperature calculation
+ * @param obj Class reference to pass all needed parameters
+ * @return Numerical value of integral and absolute error
+ */
+struct resultErrorPair Nintegrate_Outer(BounceSolution &obj);
+
+/**
+ * @brief Nderive_BounceRatio Numerical derivative for the inverse time scale
+ * calculation
+ * @param obj Class reference to pass all needed parameters
+ * @return Numerical value of derivative and absolute error
+ */
+struct resultErrorPair Nderive_BounceRatio(BounceSolution &obj);
+
+/**
+ * @brief speed of sound
+ */
+const double Csound = 0.5773502691896258; // 1/sqrt(3)
+
+/**
+ * @brief reduced Planck mass = MPl / (8 Pi)
+ */
+const double MPl = 2.4e18;
```

Comment:
Why not put them to the other const in `SMparam.h`?

Reply:
Done.